### PR TITLE
Set Up To-Do List Structure

### PR DESCRIPTION
**Pull request by angeluray**
# [To-Do List: Structure](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md)

## Hi good reviewer,

Patricio greeting

## Basic Check✔

- Created a new branch according to the Github Flow rules.
- Added a HTML file that includes HTML structure for the To-Do List web page.
- Added a CSS file that includes styles for To-Do list.
- Added a JS file that includes the logic process.
- Set up linters for HTML/CSS and JS making sure they work correctly.
- Added a .gitignore file to make sure any irrelevant files are ignored by git.
- Built using the Webpack.

## Project Requirements Basic checks ✔

- Set up a new project with webpack that is based on the webpack exercise you have already completed. - **Done.**

- Create an index.html file and write your HTML markup here. Create an empty To Do List placeholder. The index.html file must be set as a template using the HTML Webpack Plugin. - **Done.**

- Create an index.js file and set an array of some simple to do tasks (array of objects). Each task object should contain three keys:
1. description [string]. - **Done.**
2. completed [bool]. - **Done.**
3. index: [number].  - **Done.**

- Write a function to iterate over the tasks array and populate an HTML list item element for each task. - **Done.**

- On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task. - **Done.**

- Create a style.css and set rules for the To Do List. CSS must be loaded by Webpack Style/CSS Loader. Your list should be a clone of the part of the minimalist project captured in the video below. - **Done.**

- All your source files (index.html, index.js and style.css) must be located in /src directory and your distribution files will be generated by webpack and served by webpack dev server from /dist folder.  - **Done.**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+const container = document.querySelector('.myListContainer');
+container.innerHTML = '<div class="headlineContainer"><h1>Today\'s To-Do\'s</h1><img src="https://img.icons8.com/ios-glyphs/30/000000/refresh--v1.png" class="myRefresher"></div>';
+const list = document.createElement('ol');
+list.classList.add('list');
+const input = document.createElement('div');
+input.classList.add('input');
+container.appendChild(input);
+container.appendChild(list);
+list.insertAdjacentHTML('afterend', '<button type="button" id="clearAllBtn">Clear all completed</button>');
+
```
Comment (Reem-lab) [inline src/index.js:15]: - [ ] Great job so far, But according to the requirements I think you should generate array of objects contain a list so like a static list, Good job on making it dynamic but you should stick with the requirements as much as possible ✅👌🏻

- What should I see exactly just 3 or 4 tasks listed in the list 👌🏻
image

**Review by Reem-lab — CHANGES_REQUESTED** [review]
# Hi @angeluray, 👋🏻👋🏻 

Good job so far! 🔥🔥🚀🚀
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the functionality ✅
📌 Good commit messages ✅
📌 Professional README file. ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (angeluray) [reply src/index.js:15]: Hi @Reem-lab, gotcha! I will do it. I misunderstood a little bit the first project approach. Thank you.

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @angeluray 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**